docs/general-reference/settings.md

```diff
@@ -1,123 +1,87 @@
-* Test a Mobile Website by adding the Website URL, a name and the version number
-* Integrate a build plugin, like Ant, Maven, Jenkins etc.
-* Get your latest app versions directly from HockeyApp. Enter your <a href="https://rink.hockeyapp.net/manage/auth_tokens" target="_blank">HockeyApp token</a> and you are set.
-
 <h3 id="settings">Account Settings</h3>
-
-In the right corner of your screen you can administrate your Account Settings.
+To open account settings, click in the right corner of your screen, you can administrate your Account Settings.
 <img class="center shadow" src="/img/general-reference/account-settings.png">
 <br>Click on the different tabs to manage your account.</br> 
-<img class="center shadow" src="/img/settings/profile.png"> <h4 id="profile">Profile</h4>
-Here you can change your information given at your sign up, like name and email adress. What you cannot change is your user name. It will remain the same to ensure to allocate your data with your account.
+<img class="shadow" src="/img/settings/profile.png"> 
 
-<h4 id="team-accounts">Team Accounts</h4>
-If you have a Team plan, you will be able to test your companies apps in parallel with other teammates. Add and remove team members to your account.
+<h3 id="profile">Profile</h3>
+Here you can change your personal information given at your sign up, like the first name, the last name or the email address you use. But what you cannot change is your username (which can be used to login), because It is unique and must be the same without changing to ensure allocating your data with your account.
 
-- Invite your teammates by adding their email address.
-- Wait for them to accept your invitation.
-- <b>Ready!</b> Now they are able to see your apps and test results
-
-<h4 id="password">Password</h4>
+<h3 id="password">Password</h3>
 To change your password enter your new password into the text field and save. Done!<br>
 If you made a typo or forgot what you chose as a password, you can always <a href="https://app.testobject.com/#/forgot" target="_blank">set back your password</a> when you login to the TestObject cloud. 
 
 - Enter your email address or user name
 - You will receive a link via email
 - Follow this link and enter a new password.
 
-<h4 id="plans_tab">Plans</h4>
-This tab allows you to change your subscribed <a href="#plans">Testing Plans</a>. 
-
-See our homepage for an overview of all plans and our pricing: <a href="https://testobject.com/pricing" target="_blank">https://testobject.com/pricing</a>
-
-
-<h4>Billing</h4>
-
-
-Here you see your payment data and your billing history. 
-
-Update your credit card information and check all your previous payments made to TestObject.
+<h3 id="team-accounts">Inviting Team Members</h3>
+If you have a Team plan, you will be able to test your companies apps in parallel with other teammates. Add and remove team members to your account.
```

Comment:
"...you will be able to test your apps together with other teammates."
What is the "Add and remove team members to your account" phrase supposed to convey here? It seems not connected to what's around it